app/src/main/java/at/tacticaldevc/oat/utils/PrefsSupport.java

```diff
@@ -23,7 +24,7 @@
     public static Set<Contact> toContactSet(Map<String, String> contactMap) {
         ensureNotNull(contactMap, "contact Map");
 
-        Set<Contact> result = new HashSet<>();
+        Set<Contact> result = new LinkedHashSet<>();
```

Comment:
Why do you require a linked HashSet?

Reply:
Not much of a diff but leave it with HashSet<>

Reply:
Because if it wouldn't be a LinkedHashSet, I will get a new seemingly random order of Contacts.

Reply:
Tests green - Merging